docs/overview/kyuubi_vs_hive.md

```diff
@@ -32,16 +32,15 @@ have multiple reducer stages.
 
 ## Differences Between Kyuubi and HiveServer2
 
```

Comment:
I think we should remove this section. It seems a bit offensive

Reply:
Actually for me as a newbie to Kyuubi, this section helped me understand the advantages and some major differences in between